benchmark/httpapi.py

```diff
@@ -100,6 +112,104 @@ def delete(self, id):
             return fail(ResultNotFound())
 
 
+@implementer(IBackend)
+class TxMongoBackend(object):
+    """
+    The backend that uses txmongo driver to work with MongoDB.
+    """
+    def __init__(self, hostname="127.0.0.1", port=27017):
+        connection = MongoConnectionPool(host=hostname, port=port)
+        self.collection = connection.benchmark.results
```

Comment:
Does this actually make a connection? If so, then it would perhaps be better to move the connection logic to a method, and leave the constructor being super boring.